### PR TITLE
[Issue #50] Required rows check should display only one failed line

### LINKED ISSUE #50: Required rows check should display only one failed line
GitHub status description is limited to 140 characters which is not expected to be exceeded by single message. However, required rows check accumulate messages from all failed rows so we should display only one failed row message.

### PR DESCRIPTION
**Pull request by xstefank**
Issue: https://github.com/xstefank/tyr/issues/50
resolves #50

### CODE REVIEW

**Comment by xstefank** [comment]
@khermano @lukasburda can you review please?

File: src/test/java/org/xstefank/check/RequiredRowsCheckTest.java [hunk 1]
```diff
@@ -50,4 +51,19 @@ public void testIfNullParameterThrowsException() {
     public void testIfEmptyPayloadThrowsException() throws IOException {
         requiredRowsCheck.check(TestUtils.EMPTY_PAYLOAD);
     }
+
+    @Test
+    public void testMultipleFailedLines() {
+        RegexDefinition secondRow = new RegexDefinition();
+        String expectedError = "Second error message";
```
Comment (khermano) [inline src/test/java/org/xstefank/check/RequiredRowsCheckTest.java]: one little thing...please rename this..this is not and error itself :)

File: src/test/java/org/xstefank/check/RequiredRowsCheckTest.java [hunk 2]
```diff
@@ -50,4 +51,19 @@ public void testIfNullParameterThrowsException() {
...
+    @Test
+    public void testMultipleFailedLines() {
+        RegexDefinition secondRow = new RegexDefinition();
+        String expectedErrorMessage = "Second error message";
+
+        secondRow.setPattern(Pattern.compile("(?!.*)"));
+        secondRow.setMessage(expectedErrorMessage);
+
+        requiredRowsCheck = new RequiredRowsCheck(Arrays.asList(row, secondRow));
+        String errorMessage = requiredRowsCheck.check(TestUtils.TEST_PAYLOAD);
```
Comment (khermano) [inline src/test/java/org/xstefank/check/RequiredRowsCheckTest.java]: maybe I would also rename this as something more specific...e.g. actualErrorMessage :)

Comment (xstefank) [reply src/test/java/org/xstefank/check/RequiredRowsCheckTest.java]: I think that one is clear as it's directly checked on the next lines

File: src/test/resources/json/testPayload.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "pull_request": {
     "title": "Test PR",
-    "body": "Test description",
+    "body": "Test description\nNext line\n",
```
Comment (lukasburda) [inline src/test/resources/json/testPayload.json]: Github returns different line separator. `\r\n`

**Pull request merged**